test/functions

```diff
@@ -1,10 +1,10 @@
 
 test_run() {
-    echo ">>>" $* | sed -e "s!$REQFLOW!REQFLOW!" >> $T.out
+    echo ">>>" $* | sed -e "s!$REQFLOW! REQFLOW!" >> $T.out
     # separate, then merge stdout and stderr, in order
     # to have a deterministic order of lines
-    $* > $T.stdout 2>$T.stderr
+    $(eval "$* > $T.stdout 2>$T.stderr")
```

Comment:
Please do not use eval when not necessary.
I prefer something like this:

```
     # separate, then merge stdout and stderr, in order
     # to have a deterministic order of lines
-    $* > $T.stdout 2>$T.stderr
+    # and allow the execution with an exit status other than zero
+    $* > $T.stdout 2>$T.stderr || :

```

(the : is a no-operation symbol and the || catches the exit status and prevents the interruption by the shell)
(another solution would be to surround by set +e / set -e)

Reply:
The 'eval' is required here to correctly interpret the bank '!' which is a special character in this context.

And I can not replace the use of the '!' by a "|| :" at the end of the command because it does not have the same effect.
The '!' negate the exit status and the "|| :" ignore the exit status.

As I have added a requirement for 'reqflow' to return an error status in some circumstance I expect the tests of those circumstance to check that it effectively return an error.
If I discard the effect of the return status with "|| :" the test does not test the requirement anymore.
set +e / set -e as the same problem it also ignore the exit status not check for an error.